CONTRIBUTING.md

```diff
@@ -87,60 +87,105 @@ the coverage for the VirtualBox driver's package, browse to `/drivers/virtualbox
 You can hit `CTRL+C` to stop the server.
 
 ## Integration Tests
+
+### Setup
+
 We utilize [BATS](https://github.com/sstephenson/bats) for integration testing.
-This runs tests against the generated binary.  To use, make sure to install
-BATS (use that link).  Then run `./script/build` to generate the binary.  Once
-you have the binary, you can run test against a specified driver:
+This runs tests against the generated binary.  To use, first make sure to
+install BATS (use that link).  Then run `./script/build` to generate the binary
```

Comment:
I know you didn't change this, but here's a vaguely-related nitpick:

```diff
-install BATS (use that link).  Then run `./script/build` to generate the binary
+[install BATS](https://github.com/sstephenson/bats).  Then run `./script/build` to generate the binary
```

Makes it a little less ambiguous. :)